saleor/order/models.py

```diff
@@ -90,6 +90,7 @@ class Order(models.Model):
         decimal_places=settings.DEFAULT_DECIMAL_PLACES, default=0)
     discount_name = models.CharField(max_length=255, default='', blank=True)
     display_gross_prices = models.BooleanField(default=True)
+    customer_note = models.TextField(blank=True, default='')
```

Comment:
Isn't a `TextField` a bit huge?

Reply:
`Cart` also contains `note` which is later copied to `Order.customer_note` (previously was added to the list of `OrderNote`). That field is `TextField`, so I decided to keep it the same in `Order`.